src/Schema/PlasterManifest-v1.xsd

```diff
@@ -448,6 +448,31 @@
                       <xs:documentation>Specifies the value of the RootModule property.</xs:documentation>
                     </xs:annotation>
                   </xs:attribute>
+                  <xs:attribute name="copyright" type="xs:string">
```

Comment:
Backwards compatible changes to the schema (like adding new attributes), requires bumping the SchemaVersion (to 1.2 in this case) on line 13 of this file and line 76 of Plaster.psm1.  This allows Plaster to alert users who are invoking a template based on the updated schema via an older Plaster module, that they will need to update Plaster in order to process the template.